Overview.src.html

```diff
@@ -1013,9 +1063,26 @@ <h2 class="no-num no-toc">
             </li>
             <li>Return <em>P</em>.
             </li>
-            <li>If the user agent does not <span>monitor the list of available
-            presentation displays</span>, run the following steps:
+            <li>Run the following steps:
               <ol>
+                <li>If is an entry <em>(A, presentationUrl, P)</em> in the
+                <span>set of availability objects</span>, where
+                <em>presentationUrl</em> is equal to
```

Comment:
Not sure how it looks in the generated html but I think it would be hard to distinguish between presentationUrl and presentationUrl. Maybe call the first one availabilityUrl?